data/phi0474/phi012/__cts__.xml

```diff
@@ -1,17 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <ti:work xml:lang="lat" xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:latinLit:phi0474" urn="urn:cts:latinLit:phi0474.phi012" projid="latinLit:phi012">
-    <ti:title xml:lang="eng">For Rabirius on a Charge of Treason</ti:title>
+    <ti:title xml:lang="lat">Pro C. Rabirio Perduellionis Reo Ad Quirites</ti:title>
     
            <ti:edition urn="urn:cts:latinLit:phi0474.phi012.perseus-lat2" workUrn="urn:cts:latinLit:phi0474.phi012" xml:lang="lat">
                <ti:label xml:lang="lat">Pro Rabirio Perduellionis Reo Ad Quirites</ti:label>
-               <ti:description xml:lang="mul">Cicero. M. Tulli Ciceronis. Orationes, Vol. IV. Clark, Albert Curtis, editor. Oxford: Clarendon Press, 1909.</ti:description>
+               <ti:description xml:lang="mul">Cicero. M. Tulli Ciceronis. Orationes, Volume 4. Clark, Albert Curtis, editor. Oxford: Clarendon Press, 1909.</ti:description>               
```

Comment:
The title in the files is `M. Tulli Ciceronis Orationes` no period, but here is `M. Tulli Ciceronis. Orationes`

Reply:
@lcerrato you know I saw that and somehow imagined I fixed it.